AmplifyPlugins/Auth/AWSCognitoAuthPluginIntegrationTests/AuthDeleteUserTests/AuthDeleteUserTests.swift

```diff
@@ -15,10 +15,12 @@ import AmplifyTestCommon
 class AuthDeleteUserTests: AWSAuthBaseTest {
 
     override func setUpWithError() throws {
+        continueAfterFailure = false
```

Comment:
Considered putting this in `AWSAuthBaseTest.init` but didn't want to touch other tests as part of this.